config/sitemap.rb

```diff
@@ -0,0 +1,36 @@
+# not use compression
+class SitemapGenerator::FileAdapter
+  def gzip(stream, data); stream.write(data); stream.close end
+end
+SitemapGenerator::Sitemap.namer = SitemapGenerator::SimpleNamer.new(:sitemap, extension: '.xml')
+
+# default host
+SitemapGenerator::Sitemap.default_host = "https://decide.madrid.es"
```

Comment:
The default host should depend on the host defined in the Setting's variable (Setting["url"])